Formula/buildkit.rb

```diff
@@ -14,14 +15,18 @@ class Buildkit < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    doc.install %w[README.md] + Dir["docs/*.md"]
+    revision = Utils.popen_read("git rev-parse HEAD").chomp
 
-    (buildpath/"src/github.com/moby/buildkit/").install Dir["*"]
+    ldflags = %W[
+      -s -w
+      -X github.com/moby/buildkit/version.Version=#{version}
+      -X github.com/moby/buildkit/version.Revision=#{revision}
+      -X github.com/moby/buildkit/version.Package=github.com/moby/buildkit
+    ]
+    system "go", "build", "-ldflags", ldflags.join(" "), "-trimpath", "-o", bin/"buildctl"
```

Comment:
Add `-mod=vendor`, as upstream seems to make use of dependencies vendoring.

Reply:
With `-mod=vendor` the command produces a non-executable file which even not a real binary.

Reply:
That was my stupid mistake, fixed now! :)